gotestiful.go

```diff
@@ -110,7 +112,11 @@ func main() {
 				Auth: *flagAzureDevopsAuthToken,
 			},
 		})
-		if err != nil {
+
+		switch {
+		case errors.Is(err, gtf.ErrTestRunIgnore):
+			os.Exit(1) // Known error due to tests failing. No need to log.
+		case err != nil:
```

Comment:
This way we don't show an unnecessary log at the end since tests fail errors have already been printed.